Lib/test/test_list.py

```diff
@@ -157,5 +157,12 @@ class L(list): pass
         with self.assertRaises(TypeError):
             (3,) + L([1,2])
 
+    def test_preallocation(self):
```

Comment:
This should be marked as a CPython only test.

Reply:
@rhettinger Done in 7602879